tests/integration/targets/mso_schema_site_anp/tasks/main.yml

```diff
@@ -474,8 +307,9 @@
   ignore_errors: true
   register: nm_non_existing_site_template
 
-- name: Verify non_existing_site_template
+- name: Verify non_existing_site_template for NDO version > 3.7
   ansible.builtin.assert:
     that:
     - nm_non_existing_site_template is not changed
-    - nm_non_existing_site_template.msg is match("Provided site/siteId/template 'ansible_test/[0-9a-zA-Z]*/Template2' does not exist. Existing siteIds/templates{{':'}} [0-9a-zA-Z]*/Template1, [0-9a-zA-Z]*/Template2, [0-9a-zA-Z]*/Template3")
```

Comment:
Which NDO is creating an issue here?

Reply:
I added by mistake, reverted the change.